CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to Polymer
+
+There are many ways to contribute to the Polymer project! We welcome and truly appreciate contribution in all forms - issues and pull requests to the [main library](https://github.com/polymer/polymer), issues and pull requests to the [elements the Polymer team maintains](https://github.com/polymerelements), issues and pull requests to one of our many [Polymer-related tools](https://github.com/polymer), and of course we love to hear about any Polymer elements that you build to share with the community!
+
+# Logistics
+
+## Communicating with the Polymer team
+
+Beyond Github, we try to have a variety of different lines of communication open:
+
+* [Blog](https://blog.polymer-project-org)
+* [Twitter](https://twitter.com/polymer)
+* [Google+ Community](https://plus.sandbox.google.com/u/0/communities/115626364525706131031?cfem=1)
+* [Mailing list](https://groups.google.com/forum/#!forum/polymer-dev)
+* [Slack channel](bit.ly/polymerslack)
+
+## The Polymer Repositories
+
+Because of the component-based nature of the Polymer project, we tend to have lots of different repositories. Our main repository for the Polymer library itself is at [github.com/Polymer/polymer](https://github.com/polymer/polymer). File any issues or pull requests that have to do with the core library on that repository, and we'll take a look ASAP.
+
+We keep all of our element "product line" repos in the [PolymerElements](https://github.com/polymerelements) organization. For any element-specific issues or pull requests, file directly on the element's repository, such as the `paper-button` repository at [github.com/polymerelements/paper-button](https://github.com/polymerelements/paper-button).
```

Comment:
Suggest clarifying "all our elements" to "elements the Polymer team maintains and distributes"... and can we give a nod to community catalogs where other Polymer elements can be found?